cleaning_data.py

```diff
@@ -0,0 +1,199 @@
+"""change documents to UTF-8"""
+
+
+"""tohle má ostatní texty uložit do UTF-8 formátu
+Taky by to mělo vymazat všechno, kromě textu knihy ze zdrojů gutenberg"""
+"""CHANGE documents to UTF-8 and save as a new files and convert to one dictionary"""
+import os
+import glob
+
+def read_text_files_from_subfolders(main_folder_path):
+# Find all .txt files in the specified main folder and its subfolders
+    text_files = glob.glob(os.path.join(main_folder_path, '**', '*.txt'), recursive=True)
+    
+    # Dictionary to store the contents of each file
+    file_contents = {}
+    
+    # Loop through the text files, read their content, and store in dictionary
+    for i, file in enumerate(text_files):
+        with open(file, 'r',encoding='utf-8-sig') as f:
+            # Create dynamic names like 'file_1', 'file_2', etc.
+            file_contents[f'file_{i+1}'] = f.read()
+    
+    return file_contents
+
+main_folder_path = 'data'

```

Comment:
usually we use constant syntax for fixed strings, e.g. `DATA_PATH`, and we put these constant to the top of file after imports

to enable call this script from anywhere, we can work with real paths instead

```
from pathlib import Path

DATA_PATH = Path(__file__).parent / "data"
```
(__file__ is path of script in which it is written, Path is library for handling paths over operation systems, parent is folder)